### DISCUSSION TITLE
HTTP server in Coreemu

### DISCUSSION THREAD

**Discussion by north-country** [coreemu/core#876] [Q&A] [answered]
Hi everyone, I'm a newbie to coreemu.
I’m trying some simple examples to access http server but I cannot start this.
![image](https://github.com/coreemu/core/assets/35770699/2cd7ecc1-58cc-4f4a-a9a7-7a4e1865e015)

This n6 node has only two services.
![image](https://github.com/coreemu/core/assets/35770699/0b36ad99-20cb-48b6-98e8-1a9faef13e06)

When I start this simple network, i got an error that says failed to start.
![image](https://github.com/coreemu/core/assets/35770699/25da4f2e-3806-417d-8c2f-6050acb2dedc)

Probably I made some simple mistake, but I cannot fix it. please help!

**Comment by bharnden** [comment, marked as answer]
This is fixed on the develop branch with many other changes, this is just an old example service that tries to run some simple barebones web server, but is by no means something that provides any real value. Not sure what you were looking for by using it.

**Reply by north-country** [reply]
Thank you for your reply. I am mainly using CoreEmu for educational purposes to explain the basics of network technology, so it is not necessary for the environment to be realistic. I thought it would be better to have something that is easy to use. I think CoreEmu is a good teaching material because the GUI makes the topology easy to understand visually.

I was able to start the HTTP server using the deprecated "Services", but I am now having trouble because Wireshark is no longer available as a menu item.
